torch/fx/experimental/proxy_tensor.py

```diff
@@ -1153,6 +1159,9 @@ def wrap_fake(x):
             # NB: don't match on bools
             elif type(x) is int and tracing_mode == "symbolic":
                 return shape_env.create_symintnode(shape_env.create_symbol(x, source, positive=None), hint=x, source=source)
+            elif isinstance(x, (torch.ScriptObject, AbstractScriptObject)):
+                from torch._library.abstract_impl_class import create_abstract_obj
+                return create_abstract_obj(x)
```

Comment:
shouldn't this just be `isinstance(x, (torch.ScriptObject, AbstractScriptObject)`?
We shouldn't see AbstractScriptObject here right?
And we shouldn't try to construct an AbstractScriptObject from an AbstractScriptObject

Reply:
Yeah, you're right! We should forbids this.